docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -1253,12 +1253,15 @@ or being killed.
 Query Parameters:
 
 -   **dockerfile** - path within the build context to the Dockerfile. This is 
-        ignored if `remote` is specified and points to an individual filename.
+        ignored if `remote` is specified and points to an external dockerfile.
 -   **t** – repository name (and optionally a tag) to be applied to
         the resulting image in case of success
--   **remote** – A Git repository URI or HTTP/HTTPS URI build source. If the 
-        URI specifies a filename, the file's contents are placed into a file 
-		called `Dockerfile`.
+-   **remote** – A Git repository URI or HTTP/HTTPS context URI. If the
+        URI points to a single text file, the file's contents are placed into
+	a file called `Dockerfile` and the image is built with that file as the
+	context. If the URI points to a compressed tarball, the file is fetched
+	and the contents therein used as the context for the build. If `dockerfile`
+	is also specified, there must be a file with the corresponding name inside `remote`.
```

Comment:
@moysesb thanks! in that case (also see my comment https://github.com/docker/docker/pull/12099#issuecomment-105079511 below), the documentation around the CLI also needs to be updated, describing the new feature :)